test/WebJobs.Script.Tests.E2E/Functions/wwwroot/AppSettingInformation/run.csx

```diff
@@ -1,3 +1 @@
-﻿using System.Configuration;
-
-public static string Run(HttpRequestMessage req) => ConfigurationManager.AppSettings["FUNCTIONS_EXTENSION_VERSION"];
\ No newline at end of file
+﻿public static string Run(HttpRequestMessage req) => Environment.GetEnvironmentVariable("FUNCTIONS_EXTENSION_VERSION");
```

Comment:
@fabiocav, this passes now, but I'm not sure what the original test was validating, so it may need changing. Should we be able to get an `IConfiguration` from within the function now, rather than using `ConfigurationManager`?

Reply:
Not yet. Plumbing that through, so this works for now.